consumer/pdata/common.go

```diff
@@ -65,12 +66,12 @@ func (avt AttributeValueType) String() string {
 //
 //   function f1(val AttributeValue) { val.SetIntVal(234) }
 //   function f2() {
-//   	v := NewAttributeValueString("a string")
-//      f1(v)
-//      _ := v.Type() // this will return AttributeValueTypeInt
+//       v := NewAttributeValueString("a string")
+//       f1(v)
+//       _ := v.Type() // this will return AttributeValueTypeInt
```

Comment:
These are auto-generated from the cmd/pdatagen you need to change that code in order for these changes to persist.

Reply:
Thanks so much for letting me know that! I'll look into changing that.